### ISSUE TITLE
Reload translations

### ISSUE THREAD

**Issue by patspam** [wiziple/gatsby-plugin-intl#81]
I seem to have to `gatsby clean && gatsby develop` in order to get changes to my translation files (`*.json) to be picked up - is that expected, or am I doing it wrong?

Narrowing it down slightly, instead of `gatsby clean` I can do `rm .cache/redux.state` followed by a restart of `gatsby develop`

**Comment by jhoffmcd** [comment]
@patspam you are correct, see this issue: https://github.com/wiziple/gatsby-plugin-intl/issues/11. I am hoping to look into HMR for these files during development someday soon when I have a chance.

**Comment by samajammin** [comment]
I'm also having this issue. @patspam or @jhoffmcd did you ever find a solution? Thanks in advance.

**Comment by jhoffmcd** [comment]
I never ended up finding a solution, I think this is a feature that has to be implemented in the plugin but development on this plugin seems to have dropped off, unfortunately.

**Comment by wiltsu** [comment]
This issue is one of the biggest bottlenecks in our development team, takes time to clean the cache before you can see the translation changes made.
